### PR TITLE
feat: Add battery percentage property

### PR DESCRIPTION
**Pull request by pythops**

### CODE REVIEW

**Comment by surban** [comment]
Could you rename the property to `BatteryPercentage`?

**Comment by pythops** [comment]
Done

File: bluer/src/device.rs [hunk 1]
```diff
@@ -518,6 +519,13 @@ define_properties!(
                 mt
             }),
         );
+
+        /// The battery percentage of the remote device
+        property(
+            BatteryPercentage, u8,
+            dbus: (BATTERY_INTERFACE, "BatteryPercentage", u8, OPTIONAL),
```
Comment (surban) [inline bluer/src/device.rs:526]:
```suggestion
            dbus: (BATTERY_INTERFACE, "Percentage", u8, OPTIONAL),
```
In the D-Bus API it's still called `Percentage` though.

**Review by surban — CHANGES_REQUESTED** [review]

File: bluer/src/device.rs (hunk 1 above)
Comment (pythops) [reply bluer/src/device.rs]: my bad. 
It's fixed now.

**Comment by surban** [comment]
Thanks!

**Pull request merged**